pkg/k8s/apps/crud.go

```diff
@@ -48,6 +48,30 @@ func Get(ctx context.Context, dev *model.Dev, namespace string, c kubernetes.Int
 	return &StatefulSetApp{sfs: sfs}, nil
 }
 
+// Get returns the dev pod for an app and loops until it success
+func Get(ctx context.Context, namespace, appName string, c kubernetes.Interface) (App, error) {
```

Comment:
Why is this function changing? If I read the diff correctly you've renamed `Get` to `GetByDev` but then this method looks modified. Who's calling it now?

Reply:
this new method is used by the global forwarder to get an app given a name instead of get it using the dev info (model.Dev), for that reason I distinguished between GetByDev and Get, maybe I should rename Get to GetByName?

Reply:
You are right @jmacelroy. **Get** was a new function to get an app by name instead of using model.Dev but in the end it is not necessary and stayed that way. I reverse the changes related to it.